recipes-bsp/imx-sc-firmware/imx-sc-firmware_1.5.0.bb

```diff
@@ -1,22 +1,26 @@
 # Copyright (C) 2016 Freescale Semiconductor
-# Copyright (C) 2017-2019 NXP
+# Copyright 2017-2019 NXP
```

Comment:
Please add (C) and also adjust to 2020.

Reply:
I went and actually adjusted all *Copyright* notes in recipes I touched so far.